src/Routes.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-
 import Nav from '../src/components/Nav/Nav';
 import Main from '../src/pages/Main/Main';
+import GosuDetail from '../src/pages/GosuDetail/GosuDetail';
 
 function Routes() {
   return (
     <Router>
       <Nav />
       <Switch>
-        <Route exact path="/" component={Main} />
+        <Route exact path="/main" component={Main} />
```

Comment:
```suggestion
        <Route exact path="/" component={Main} />
```
"/main" 보다는 "/"이 메인 페이지에 어울리는 경로 아닐까요?